llvm/test/CodeGen/AIE/aie2/hardware-loops/loop-with-call.ll

```diff
@@ -0,0 +1,132 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+
+; This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+; See https://llvm.org/LICENSE.txt for license information.
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+;
+; (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+
+; RUN: llc -O2 -mtriple=aie2 --enable-aie-hardware-loops \
+; RUN:   --enable-aie-zero-overhead-loops %s -o - | FileCheck %s
```

Comment:
I'd be happy to have those tests added earlier as baseline, so we can see how they get updated with each commit.